pyV2DL3/eventdisplay/IrfInterpolator.py

```diff
@@ -66,13 +66,20 @@ def __load_irf(self):
                 irf_axes[i] = np.concatenate(
                     (axis.flatten(), axis.flatten() + 0.01), axis=None
                 )
-        self.irf_data = irf_data
+
+        irf_axes[-2] = np.cos(np.radians(irf_axes[-2]))[::-1]
```

Comment:
So the second last axes is always and per definition the zenith axes? Never something else?

Reply:
That's right. The second last axis is always Zenith axis. For 1D irf axes are (energy, pedvar, zenith, camera offsets) and for 2D irf axes are (energy, rad, pedvar, zenith, camera offsets).

Reply:
There must be another possibility that we actually read out and check that this is the zenith axis. Nobody will remember this implementation when there is a change in the GDAF. Is there no way to find out what the axis type is?

Reply:
These axes are read from eventdisplay's irf ROOT file, right? I am not sure if these axes names are stored somewhere! If they are stored then of course we can find out the axis types.

Reply:
That's true, it is from the root file. But then you can simply check the axis name.